src/components/header/header.css

```
@@ -78,6 +83,13 @@
   background: rgba(242, 242, 247, 0.5);
 }
 
+.darkSearch {
+  background-color: rgba(22, 22, 34, 0.979);
+}
+.darkSearch input {
```

Please, make sure to leave a blank line between every selector. Thank you!

```suggestion
}

.darkSearch input {
```